armada/armada.py

```diff
@@ -24,16 +24,14 @@ def __init__(self, args):
 
         self.tiller = Tiller()
 
-
     def find_release_chart(self, known_releases, name):
         '''
         Find a release given a list of known_releases and a release name
         '''
-        for chart_name, version, chart, values in known_releases:
+        for chart_name, _, chart, values in known_releases:
```

Comment:
since the variable is not breaking anything, it will be okay leaving it there. however we should change it to `_version` since the values its still not being used and is internal to the class for now.